english/cheddar/Land-Of-Cheddar.md

```diff
@@ -0,0 +1,16 @@
+Ahh, yes. The vast golden land. 
+
+The world that has enveloped your every thought Every wish. Dream. Desire.
+
+Cheddar.
+
+It's so very good to be here.
+
+
+But, sadly, you must leave.
+
+Where do you travel to next?
+
+[To grab some coffee](../coffee/coffee.md)
+
+[To buy a GameBoy](../buy-gameboy/gameboy.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.